### PR TITLE
add multisite translation keys

### PR DESCRIPTION
**Pull request by ETLaurent**
## Summary

* Add translation keys used by the multisite pro module.

cf https://github.com/apostrophecms/multisite/pull/107

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [x] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [x] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue [hunk 1]
```diff
@@ -38,7 +38,7 @@
               <AposIndicator
                 icon="help-circle-icon"
                 class="apos-field__help-tooltip__icon"
-                :tooltip="field.help || field.htmlHelp"
+                :tooltip="help"
```
Comment (ETLaurent) [inline modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue:41]: was never translated, it seems!

File: modules/@apostrophecms/schema/ui/apos/logic/AposInputWrapper.js [hunk 2]
```diff
@@ -67,6 +67,26 @@ export default {
...
+    // Build interpolation object when a translation key contains interpolations
+    // following this pattern: "aTranslationKey|foo=bar|fizz=buzz".
+    // i18n will be given the "aTranslationKey" key with following object:
+    // { foo: "bar", fizz: "buzz" }
+    //
+    // Usage example:
+    //
+    // ```
+    // help: `aposMultisite:shortNameHelp|baseDomain=${self.baseDomain}`
+    // ```
```
Comment (ETLaurent) [inline modules/@apostrophecms/schema/ui/apos/logic/AposInputWrapper.js:79]: @boutell what do you think? I tried to come up with a simple solution

Comment (ETLaurent) [reply modules/@apostrophecms/schema/ui/apos/logic/AposInputWrapper.js:79]: cf https://github.com/apostrophecms/multisite/pull/107/files#diff-514affe4dea6930297c29e391064e783ad8dabe940ed8a9f3b79369efb97b786R36

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Add translation keys used by the multisite pro module.
+* Add the possibility to set dynamic values in the fields' `help` and `htmlHelp` attributes following this pattern: `translationKey|foo=bar|fizz=buzz`,
```
Comment (ETLaurent) [inline CHANGELOG.md:8]: cf https://github.com/apostrophecms/apostrophe/pull/4437/files#r1502556802

will need a cypress test if we continue with this solution

File: modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue [hunk 4]
```diff
@@ -38,7 +38,7 @@
               <AposIndicator
                 icon="help-circle-icon"
                 class="apos-field__help-tooltip__icon"
-                :tooltip="field.help || field.htmlHelp"
+                :tooltip="$t(field.help || field.htmlHelp)"
```
Comment (ETLaurent) [inline modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue:41]: was not translated

Comment (boutell) [reply modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue:41]: !! Good catch

**Review by boutell — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Add translation keys used by the multisite pro module.
```
[the remark this answers is not in the document]
Comment (boutell) [reply CHANGELOG.md:7]: Please add a changelog entry about the fix to allow localization of `help` and `htmlHelp`.

**Review by boutell — CHANGES_REQUESTED** [review]

**Review by boutell — APPROVED** [review]

**Pull request merged**